README.md

```diff
@@ -1 +1,38 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+> This is my first project at Microverse. The starting point of the great adventure.
+
+
+## Built With
+
+- HTML/CSS
+- Git & GitHub
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Samuellekpor)
+- Twitter: [@twitterhandle](https://twitter.com/samuel_lekpor)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sammuel-lekpor)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job using the correct readme template 💯, but your licence link is broken , kindly fix it by changing `[MIT](./LICENSE)` to `[MIT](./MIT.md)`